source/install/docker-upgrading.rst

```diff
@@ -1,41 +1,41 @@
 .. _update-server.docker:
```

Comment:
This file has been completely reworked (see https://jira.percona.com/browse/PMM-6275)